lib/mwUtil.js

```diff
@@ -301,14 +301,31 @@ mwUtil.createRelativeTitleRedirect = (path, req, options) => {
             .replace(pathBeforeTitle, '');
     const pathSuffixCount = (pathSuffix.match(/\//g) || []).length;
     const backString = Array.apply(null, { length: pathSuffixCount }).map(() => '../').join('');
+    let location;
     if (!options.dropPathAfterTitle) {
         const pathPatternAfterTitle = path.substring(path.indexOf('{title}') - 1);
-        return backString
+        location = backString
```

Comment:
@gwicke The semantics is different the `prefixPath`. `backString` contains the `../../` elements exactly the number of time we need to go back from the current URI. `prefixPath` feels kinda different to me.